backend/src/main/java/com/tourplanner/backend/controller/impl/TourController.java

```diff
@@ -0,0 +1,51 @@
+package com.tourplanner.backend.controller.impl;
+
+import com.tourplanner.backend.controller.IGenericController;
+import com.tourplanner.backend.service.dto.TourDTO;
+import com.tourplanner.backend.service.IGenericService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/tour")
```

Comment:
The controllers are great, but I guess we need to stick to REST guidelines when it comes to naming endpoints. Like here, the general request mapping should be "/api/v1/tours".
For your reference: [guide](https://restfulapi.net/resource-naming/), and picture:
![image](https://github.com/imoalim/TourPlanner/assets/127953460/36e33e30-ca5f-4ec1-b0de-7cba52903acb)